propkatraj/tests/test_regression.py

```diff
@@ -170,56 +130,17 @@ def test_start_stop_step(tmpdir, u, start, stop, step):
         assert_almost_equal(pkas.index.values, times, decimal=5)
 
 
-@pytest.mark.parametrize('start, stop, step', [
-    (None, 5, None), (None, None, 25)
-])
```

Comment:
Did you remove it because @IAlibay said that there's no need to check AnalysisBase functionality?

Reply:
This shouldn't get removed, the def_start_stop_step above should instead.

Reply:
@IAlibay I'm renaming the tests as I go as well since the AnalysisBase is now the only supported option. The test above was changed to test the same thing as the one I removed.

Reply:
It leads to a very confusing diff unfortunately, but yeah ok that makes sense :(